Branch-TestBed/Branch-SDK test/Branch_SDK_Debugger_Tests.m

```diff
@@ -0,0 +1,65 @@
+//
+//  Branch_SDK_Debugger_Tests.m
+//  Branch-TestBed
+//
+//  Created by Qinwei Gong on 2/23/15.
+//  Copyright (c) 2015 Branch Metrics. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import <XCTest/XCTest.h>
+#import "BNCPreferenceHelper.h"
+#import "BranchServerInterface.h"
+#import "Nocilla.h"
+
+@interface Branch_SDK_Debugger_Tests : XCTestCase {
+    
+@private
+    BNCPreferenceHelper *prefHelper;
+    BranchServerInterface *serverInterface;
+}
+
+@end
+
+@implementation Branch_SDK_Debugger_Tests
+
+- (void)setUp {
+    [super setUp];
+    
+    [[LSNocilla sharedInstance] start];
+    
+    prefHelper = [[BNCPreferenceHelper alloc] init];
+    serverInterface = [[BranchServerInterface alloc] init];
+    serverInterface.delegate = (id<BNCServerInterfaceDelegate>)prefHelper;    
+}
+
+- (void)tearDown {
+    [[LSNocilla sharedInstance] clearStubs];
+    [[LSNocilla sharedInstance] stop];
+    
+    [super tearDown];
+}
+
+- (void)testConnectSucceed {
```

Comment:
I'm not sure I understand this (I still need to checkout this Branch locally to see how it runs instead of just examining code), but it seems that you've created two targets, and now have code duplicated between both. Is that correct? And if so, can it be removed?